homeassistant/components/switchbot_cloud/__init__.py

```diff
@@ -72,6 +73,10 @@ def make_device_data(
             devices_data.switches.append(
                 prepare_device(hass, api, device, coordinators_by_id)
             )
+        if isinstance(device, Device) and device.device_type == "MeterPlus":
```

Comment:
I think Meter, MeterPlus and Outdoor meter can all be handled the same way looking at the API. Can we handle the 3 types ?

Reply:
Yes good call! I've added support for all three. They essentially behave the same so we can treat them all the same.

Reply:
Oh, good idea